Remotion/ObjectBinding/Web.Development.WebTesting.TestSite/Program.cs

```diff
@@ -9,6 +9,8 @@ public class Program
   {
     public static void Main (string[] args)
     {
+      LegacyAspNetInitialization.License = "I hereby confirm that I use CoreForms only as part of Software developed by RUBICON IT GmbH.";
```

Comment:
Please switch to the trial-string. am not sure which would be best, but it feels like that's the way to go for non-prod strings